code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -521,33 +521,7 @@
 			air.set_temperature(clamp(thermal_energy/new_heat_capacity, TCMB, INFINITY)) //THIS SHOULD STAY OR FUSION WILL EAT YOUR FACE
 		return REACTING
 
-/datum/gas_reaction/fusion/test()
-	var/datum/gas_mixture/G = new
-	G.set_moles(GAS_CO2,300)
-	G.set_moles(GAS_PLASMA,1000)
-	G.set_moles(GAS_TRITIUM,100.61)
-	G.set_moles(GAS_NITRYL,1)
-	G.set_temperature(15000)
-	G.set_volume(1000)
-
-	var/result = G.react()
-	if(result != REACTING)
-		return list("success" = FALSE, "message" = "Reaction didn't go at all!")
-
-	var/instability	= G.analyzer_results["fusion"]
-	var/plas = G.get_moles(GAS_PLASMA)
-	var/co2 = G.get_moles(GAS_CO2)
-	var/temp = G.return_temperature()
-
-	if(abs(instability - 2.66) > 0.01)
-		return list("success" = FALSE, "message" = "Fusion is not calculating analyzer results correctly, should be 2.66, is instead [instability]")
-	if(abs(plas - 458.241) > 0.5)
-		return list("success" = FALSE, "message" = "Fusion is not calculating plasma correctly, should be 458.241, is instead [plas]")
-	if(abs(co2 - 505.369) > 0.5)
-		return list("success" = FALSE, "message" = "Fusion is not calculating co2 correctly, should be 505.369, is instead [co2]")
-	if(abs(temp - 112291) > 200) // I'm not calculating this at all just putting in the values I get when I do it now
-		return list("success" = FALSE, "message" = "Fusion is not calculating temperature correctly, should be around 112291, is instead [temp]")
-	return ..()
+//has fusion ever worked?
```

Comment:
Yes. It has. You should make a pr completely removing it and associated systems and pr it before you do this

Reply:
it's a mechanic as opposed to something like the gas formations, which are just interactions.